app/server/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "lint": "eslint . --cache",
     "prebuild": "yarn && rm -rf build",
-    "build": "babel-node ../node_modules/.bin/webpack",
+    "build": "babel-node --presets es2015 $(npm bin)/webpack",
```

Comment:
The addition of the `es2015` preset flag in the CLI isn't comparable to setting the same config in `.babelrc`. Whilst both the UI and server bundles are getting generated, we're now getting runtime errors relating to `import` directives in both bundles not being transpiled. This is because setting the `es2015` preset flag only applies to the `babel-node` runtime (needed for the `import` directives in the Webpack config files); it doesn't get propagated to the `babel` Webpack loader runtime, since they're too distinct things.

What we're needing to transpile right now is minimal - the `import` to commonJS syntax is the only thing, so I'd suggest adding the same preset inline in the Webpack config files too.